projects/fsastorefrontlib/src/core/global-message-config/default-global-message-config.ts

```diff
@@ -0,0 +1,20 @@
+import { GlobalMessageConfig, GlobalMessageType } from '@spartacus/core';
+
+export function defaultFSGlobalMessageConfigFactory(): GlobalMessageConfig {
+  return {
+    globalMessages: {
+      [GlobalMessageType.MSG_TYPE_CONFIRMATION]: {
+        timeout: 15000,
```

Comment:
Timeout configuration (delay before closing global message) is:
- Confirmation: 3s
- Info: 3s 
- Warning: 7s

It is really hard to catch message, especially in case of confirmations and infos.
- Error: 7s